services/f-droid/f-droid.tester.js

```diff
@@ -0,0 +1,35 @@
+
+t.create('Package is found')
+  .get('/version/org.pacien.tincapp.json')
+  .intercept(nock =>
+    nock('https://f-droid.org')
+      .get('/en/packages/org.pacien.tincapp/')
+      .reply(200, manyVersions)
```

Comment:
Could we not make this test hit the real service? It will be easier to spot changes to the website that way. 😉

Reply:
I wonder: If apps change their version, the test will fail and other developers do not know how to handle this.
I would create another test case to test if the API does not change:

- invalid app id
- valid app id should have version result, not "fix this badge" but `/[0-9]+(\.[0-9]+)*/` or a number in it.

Reply:
e487bfb would be such a test where I look that the value is valid and the API did not change. But: I am not able to write such a test right now.

Reply:
I got it to work.